File: Pruebas/Proyecto de Afiliacion/ApiAppAfiliacion/Models/Compra.cs
namespace ApiAppAfiliacion.Models
{
    public class Compra
    {
        public int Id { get; set; }
        public string NombreCliente { get; set; }
        public string Ubicacion { get; set; }
        public DateTime FechaHora { get; set; }
        public string? CodigoDescuento { get; set; }

    }
}

File: Pruebas/Proyecto de Afiliacion/Afiliacion/app/src/main/java/com/nuevocarnic/afiliacion/Home/Compra.kt
package com.nuevocarnic.afiliacion.Home

data class Compra(
    val id: Int,
    val nombreCliente: String,
    val ubicacion: String,
    val fechaHora: String, // ISO-8601 del backend. Usás String o LocalDateTime si lo parseás.
    val codigoDescuento: String? = null
)